next.config.js

```diff
@@ -2,9 +2,27 @@
 const nextConfig = {
   reactStrictMode: true,
   compiler: { styledComponents: true },
+  eslint: {
```

Comment:
This is the root of the problem with eslint / prettier not picking stuff up. Nextjs only lints certain dirs by default. Added other dirs here and now it catches the missing issues.